include/secp256k1_schnorrsig.h

```diff
@@ -0,0 +1,125 @@
+#ifndef SECP256K1_SCHNORRSIG_H
+#define SECP256K1_SCHNORRSIG_H
+
+#include "secp256k1.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** This module implements a variant of Schnorr signatures compliant with
+ * BIP-schnorr
+ * (https://github.com/sipa/bips/blob/bip-schnorr/bip-schnorr.mediawiki).
```

Comment:
```suggestion
 * (https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki).
```
This link is broken until [the PR is merged](https://github.com/bitcoin/bips/pull/876).